coderd/wsbuilder/wsbuilder_test.go

```diff
@@ -773,6 +774,67 @@ func TestWorkspaceBuildWithRichParameters(t *testing.T) {
 	})
 }
 
+func TestWorkspaceBuildWithPreset(t *testing.T) {
```

Comment:
For completeness, we might want another test here which validates the `IsPrebuild` field as well.